catalog/spatial/registry/registry-source-configuration-handler/src/main/java/org/codice/ddf/registry/source/configuration/SourceConfigurationHandler.java

```diff
@@ -370,6 +373,12 @@ private void deleteRegistryConfigurations(Metacard metacard)
                 RegistryObjectMetacardType.REGISTRY_ID,
                 registryId));
         if (configurations != null) {
```

Comment:
hmm, that is a bit of a tough one. Might still want to check it in case the implementation doesn't follow it. Since you are accessing 0 element might be worth the minimal cost of checking the size. Open to other thoughts on it.